tests/python/test_scene.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 import pytest
 
+from _helios import HeliosException
```

Comment:
I am wondering whether we should expose `HeliosException` in the package instead. If a user wanted to catch it, they should not need to pull it from the `_helios` package.